cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/flow/FlowletRuntimeService.java

```diff
@@ -68,7 +69,13 @@ protected void startUp() throws Exception {
 
     serviceHook.startAndWait();
     initFlowlet();
-    flowletProcessDriver.startAndWait();
+    try {
+      flowletProcessDriver.startAndWait();
+    } catch (UncheckedExecutionException exception) {
+      throw exception;
```

Comment:
Right, but in case of no exception, there is a new successfully "started" flowlet log.